examples/vanilla/control-elements/media-error-dialog.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Media Error Dialog</title>
+    <script type="module" src="../../../dist/index.js"></script>
+    <script type="module">
+      import { CustomVideoElement } from 'https://cdn.jsdelivr.net/npm/custom-media-element/+esm';
+
+      class CustomVideo extends CustomVideoElement {
+        #error;
+
+        get error() {
+          return this.#error ?? super.error;
+        }
+
+        set error(value) {
+          this.#error = value;
+        }
+      }
+
+      globalThis.customElements.define('custom-video', CustomVideo);
+    </script>
+
+    <style>
+      /* Hide custom elements that are not defined yet */
+      :not(:defined) {
+        display: none;
+      }
+
+      h2 {
+        margin-top: 2rem;
+      }
+
+      /** add styles to prevent CLS (Cumulative Layout Shift) */
+      media-controller:not([audio]) {
+        display: grid;          /* expands the container if preload=none */
+        max-width: 540px;       /* allows the container to shrink if small */
+        aspect-ratio: 16 / 9;   /* set container aspect ratio if preload=none */
+      }
+
+      video,
+      img[slot="poster"] {
+        grid-area: 1 / 1;     /* position poster img on top of video element */
+        width: 100%;          /* prevents video to expand beyond its container */
+      }
+
+      .examples {
+        margin-top: 20px;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>MediaErrorDialog</h1>
+
+    <h2>Non existing mp4</h2>
+
+    <media-controller defaultsubtitles>
+      <custom-video
+        slot="media"
+        src="https://stream.mux.com/Sc89iWAyNkhJ3P1rQ02nrEdCFTnfT01CZ2KmaEcxXfB00/low.mp4"
+        muted
+        crossorigin
+      ></custom-video>
+      <img slot="poster" src="https://image.mux.com/Sc89iWAyNkhJ3P1rQ02nrEdCFTnfT01CZ2KmaEcxXfB008/thumbnail.webp?time=13" />
+      <media-loading-indicator slot="centered-chrome" noautohide></media-loading-indicator>
+      <media-error-dialog hidden></media-error-dialog>
```

Comment:
I don't love that `hidden` has to be added by default. it was used on `media-chrome-dialog` similar to `menu` but maybe an `open` attribute like the native dialog could be nicer here.

Reply:
Yeah modeling it more like https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog feels right to me.